platform-sdk/docs/proposals/001-tss-rosters-design-proposal/proposal.md

```diff
@@ -278,10 +299,9 @@ A Roster is considered valid if it satisfies the following conditions:
 6. All ServiceEndpoint/s must have a valid IP address or domain name (mutually exclusive), and port.
 7. The roster must have a unique NodeId for each RosterEntry.
 
-On the submission of a new Candidate Roster, the platform will validate the roster against these conditions. Note
-that a constructed `Roster` can be valid, but not accepted by the platform. For example, if a new candidate roster
-is set via the API, but its hash evaluates to the hash of the existing candidate roster, the new roster will be
-discarded. That is, the operation has no effect.
+On the submission of a new Candidate Roster, the platform will validate the roster against these conditions.
+Note that a constructed `Roster` can be valid, but not accepted by the platform.
```

Comment:
Similar to above, I suggest to re-evaluate usages of the word "platform" in this document to align it with the current (and future) state of the architecture.